specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-preview/billing.json

```diff
@@ -0,0 +1,9330 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/enrollmentAccounts/{enrollmentAccountName}/billingRoleAssignments/{billingRoleAssignmentName}": {
+      "get": {
+        "tags": [
+          "BillingRoleAssignments"
+        ],
+        "x-ms-examples": {
+          "EnrollmentAccountRoleAssignment": {
+            "$ref": "./examples/EnrollmentAccountRoleAssignment.json"
+          }
+        },
+        "operationId": "BillingRoleAssignments_GetByEnrollmentAccount",
+        "description": "Gets a role assignment for the caller on a enrollment Account. The operation is supported only for billing accounts with agreement type Enterprise Agreement.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/enrollmentAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/billingRoleAssignmentNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingRoleAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "BillingRoleAssignments"
+        ],
+        "x-ms-examples": {
+          "EnrollmentAccountRoleAssignmentDelete": {
+            "$ref": "./examples/EnrollmentAccountRoleAssignmentDelete.json"
+          }
+        },
+        "operationId": "BillingRoleAssignments_DeleteByEnrollmentAccount",
+        "description": "Deletes a role assignment for the caller on a enrollment Account. The operation is supported only for billing accounts with agreement type Enterprise Agreement.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/enrollmentAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/billingRoleAssignmentNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingRoleAssignment"
+            }
+          },
+          "204": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingRoleAssignment"
```

Comment:
Delete mustnot return a response body. It should be empty. A bunch of 204  response codes are added in this PR all of which have a response body. Existing 200 OK responses also have a body. This is not compliant with Azure Resource Provider contract.

Reply:
this was a accidental change reverted.